pkg/server/server.go

```diff
@@ -0,0 +1,47 @@
+package server
+
+import (
+	"crypto/tls"
+	"fmt"
+	"net/http"
+	"recommends/pkg/config"
+	"time"
+
+	klog "k8s.io/klog/v2"
+
+	"github.com/gorilla/mux"
+)
+
+func StartAndListen() {
+	port := config.Cfg.HttpPort
+
+	// Configure TLS
+	cfg := &tls.Config{
+		MinVersion:               tls.VersionTLS12,
+		CurvePreferences:         []tls.CurveID{tls.CurveP521, tls.CurveP384, tls.CurveP256},
+		PreferServerCipherSuites: true,
+		CipherSuites: []uint16{
+			tls.TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384,
+		},
+	}
+
+	// Initiate router
+	router := mux.NewRouter()
+	router.HandleFunc("/liveness", livenessProbe).Methods("GET")
+	router.HandleFunc("/readiness", readinessProbe).Methods("GET")
+	router.HandleFunc("/computeRecommendation", readinessProbe).Methods("POST")
```

Comment:
@Anxhela21 will update this method with actual implementation for computeRecommendation @SherinV